foundry/foundry.py

```diff
@@ -618,12 +618,26 @@ def _load_data(self, file=None, source_id=None, globus=True):
                 self.dataset.dataframe = pd.read_json(
                     os.path.join(path, file)
                 )
-            except:
-                # Try to read individual lines instead
-                self.dataset.dataframe = pd.read_json(
-                    os.path.join(path, file), lines=True
-                )
-
+            except Exception as e:
+                print("Reading {} as JSON failed: {} \n".format(file, e), "Now attempting to read as JSONL")
+
+            if (self.dataset.dataframe is None):
```

Comment:
currently, this code will try to read the file each of these ways every time, which we dont necessarily want -- if it works one way, we want to stop trying

Reply:
so instead of having separate blocks, I think this can all be condensed into slightly cleaner code by doing some sort of exception chaining? I'm trying to think of the best way to do it, but I think it'd require a little playing around here. something like 
```
try: 
    one way
    except:
         print that this way failed, what you're about to try
         try:
               second way
         except:
               print that second way failed
               try:
                    etc
```
I believe you or Zoa originally had this in the other PR, and I suggested one `try` with multiple `excepts`, but after further reading I realize that doesn't actually work as I expected (my bad!). 

also, why are you checking `if (self.dataset.dataframe is None)`, could you clarify the logic here?

Reply:
something like this: https://stackoverflow.com/questions/17322208/multiple-try-codes-in-one-block

Reply:
@ascourtas, the `if (self.dataset.dataframe is None)` is for that, if one of the readings works, so if it is read as a JSON successfully, then the statement evaluates to false and doesn't attempt any of the other readings -- I was able to check this in testing and it was working as intended, so when a certain type of reading fails the dataframe is still none, but once it works it is no longer none and doesnt attempt another read-- but we can write the nesting way as earlier if that would be better

Reply:
@ribhavb ahhh I see! that makes sense. I feel like there might be a way to make it more concise, but I cant think of it off the top of my head. I say this is good for now, and we can always refactor in the future if we wanna